Formula/node@14.rb

```diff
@@ -22,17 +23,56 @@ class NodeAT14 < Formula
 
   depends_on "pkg-config" => :build
   depends_on "python@3.9" => :build
+  depends_on "brotli"
+  depends_on "c-ares"
   depends_on "icu4c"
+  depends_on "libnghttp2"
+  depends_on "libuv"
```

Comment:
We should check linkage with `libuv` when the bottles are built. Node had a bug where it didn't link properly with libuv (#76893). Not sure if the fix was ever backported.

Reply:
Big Sur arm64 and x86_64 bottles:

```
bottles/node@14/14.18.0_1/bin/node:
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	@@HOMEBREW_PREFIX@@/opt/libuv/lib/libuv.1.dylib (compatibility version 2.0.0, current version 2.0.0)
```

I think that means its ok?

Reply:
Yea, that's fine.